app/src/components/Preferences/PreferencesCard.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Card,
+  CardContent,
+  Typography,
+  LocationOnIcon,
+} from '../../store/index';
+import { useStyles } from './PreferencesCardStyles';
+
+export default function PreferencesCard() {
+  const classes = useStyles();
+  return (
+    <Card className={classes.root}>
+      <CardContent>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}>
+          <Typography variant='h4'>Training</Typography>
+          <Typography variant='h6' className={classes.subBody}>
```

Comment:
If you add a prop to Typography of `color="primary"` then you don't need this `classes.subBody` className and you use the colors in the theme object.